apollo-federation-types/Cargo.toml

```diff
@@ -26,7 +26,7 @@ apollo-federation = { workspace = true }
 
 # config and build dependencies
 serde = { version = "1", features = ["derive"] }
-schemars = { version = "1.0.0", optional = true, features = ["url2"] }
+schemars = { version = "1.0.4", optional = true, features = ["url2"] }
```

Comment:
Do we really need to pin it down to the patch version? Why not `"1"` or `"1.0"` here?

Reply:
Hey @duckki, I didn't mean it that way. Sorry, I'm new to Cargo 😅. I've changed it to "1" now. Thanks for the feedback! 🙏

Reply:
It was a genuine question. So, I don't know which is the right one. In general, I try not to pin too specifically, but there are exceptions.

Reply:
Specifying 1.0.4 means "latest 1.x version, but at least 1.0.4", so it's not really that different from specifying `1` or `1.0.0`. So it's not _pinned_ unless you write `=1.0.4`. In most cases it's fine to just specify the most recent available version, if you're working on a frequently used library then you might want to be more careful about actually specifying the minimum version you support.

Reply:
yeah, both are totally fine here 😁